File: Lista2/Python/Lsala2/Ex1.py
n1 = float(input('Nota 1: '))
n2 = float(input('Nota 2: '))
n3 = float(input('Nota 3: '))
media = (n1 + n2 + n3)/3
if media >= 6:
    print(f'Aluno aprovado com média {media:.2f}')
else:
        print(f'Aluno reprovado com média {media:.2f}')

File: Lista2/C/Lsala2/Ex1.c
#include <stdio.h> 

int main()
{
	float n1, n2, n3, media;
	printf("Nota 1: ");
	scanf("%f", &n1);
	printf("Nota 2: ");
	scanf("%f", &n2);
	printf("Nota 3: ");
	scanf("%f", &n3);
	media = (n1 + n2 + n3) / 3;
	if(media >= 6){
		printf("Aluno aprovado com media %.1f", media);
	}else{
		printf("Aluno reprovado com media %.1f", media);
	}
}


